kernel/context.mli

```diff
@@ -160,6 +160,9 @@ sig
       Outermost declarations are processed first. *)
   val fold_outside : (('c, 't) Declaration.pt -> 'a -> 'a) -> ('c, 't) pt -> init:'a -> 'a
 
+  (** Return the set of all named variables bound in a given rel-context. *)
+  val ids : ('c, 't) pt -> Id.Set.t
```

Comment:
Sorry to nitpick again, but a three-letter function in a global API is looking for trouble. There are many parts of the code that use the `ids` string as a variable and this will create weird interactions. Why don't you also call it `to_vars` by analogy with the named case?

Reply:
For `named_context`, all variables are named, so `to_vars` makes sense suggesting a total map. In the `rel_context` case, this collects only the `Name`s. In practice, it is supposed to be qualified as in `Context.Rel.ids`. A longer name could otherwise be `ids_of`, or, to be closer to `to_vars`, `to_named_vars`, or even `named_vars`?

Reply:
I don't think that `to_*` indicates a total map. Case in point, `string_to_int`.

Reply:
OK, I used `to_vars`.